.github/workflows/ci-python.yml

```diff
@@ -0,0 +1,42 @@
+name: Python CI
+
+on:
+  pull_request:
+    paths:
+      - packages/@n8n/task-runner-python/**
+      - .github/workflows/ci-python.yml
+  push:
+    branches:
+      - master
```

Comment:
We probably don't need to run it again in the master branch do we? I think we only do that in the main CI because we test on node 20/22/24 etc.